docs/build/windows.md

```diff
@@ -341,6 +346,17 @@ To include UserDefines.mak in the build, use the command line parameter
 `-DUSERDEFINES`. You can also set an environment variable `USERDEFINES=1` to get
 the same result.
 
+### JAVA_HOME
+
+This environment variable tells Gradle what version of Java to use for building Keyman for Android.
+OpenJDK 8 is used to build stable-16.0 while OpenJDK 11 is used for master.
+It's recommended to install both and set the environment variables to:
+```bat
+SET JAVA_HOME_8="path to OpenJDK 8"
+SET JAVA_HOME_11="path to OpenJDK 11"
+SET JAVA_HOME="$JAVA_HOME_8"
+```
```

Comment:
````suggestion

**Multiple versions of Java:** If you need to build Keyman for Android 16.0 or older versions, you can set `JAVA_HOME_11` to the OpenJDK 11 path and `JAVA_HOME` to the OpenJDK 8 path. This will build both versions correctly from command line. But note that you do need to update your `JAVA_HOME` env var to the associated version before opening Android Studio and loading any Android projects. `JAVA_HOME_11` is mostly used by CI.

```bat
SET JAVA_HOME="path to OpenJDK 8"
SET JAVA_HOME_11="path to OpenJDK 11"
```
````